pkg/core/coretesting/expandable_resource.go

```diff
@@ -10,7 +10,7 @@ import (
 
 type CreateCase[P any, R core.ExpandableResource[P]] struct {
 	Name     string
-	Existing R
+	Existing []core.Resource
```

Comment:
why change this to an array?

Reply:
The more significant bit of the change is that it goes from an `R` to any `core.Resource`. I use this to test an S3Object.Create when there's already an existing S3Bucket. I thought I may need it as an array too, and though I didn't end up needing that, I figured I'd keep it since I'd already done it.

Do you prefer it as single?

Reply:
yeah i think so just because thats how all the tests are structured (even though they dont all use the testing utility yet)